x-pack/plugin/sql/sql-proto/src/main/java/org/elasticsearch/xpack/sql/proto/Protocol.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+package org.elasticsearch.xpack.sql.proto;
+
+import org.elasticsearch.common.unit.TimeValue;
+
+import java.util.TimeZone;
+
+/**
+ * Sql protocol defaults and end-points shared between JDBC and REST protocol implementations
+ */
+public final class Protocol {
+    public static final TimeZone TIME_ZONE = TimeZone.getTimeZone("UTC");
+
+    /**
+     * Global choice for the default fetch size.
+     */
+    public static final int FETCH_SIZE = 1000;
```

Comment:
I just moved them from the original request here since we now have 2 requests. We can address this, but I think it should be in another PR, since this will change the protocol as well (these parameters are used not only for the initial client config, but also on the server side as part of the protocol. For example, we don't serialize these parameters if they have default values.)